### DISCUSSION TITLE
Folder/Tab options in app drawer

### DISCUSSION THREAD

**Discussion by Spoonmugen** [LawnchairLauncher/lawnchair#3338] [Q&A] [answered]
been waiting few builds for folder/tabs to be added back like lawnchair 2.1

cant alpha/beta test without this as this was a huge benefit of using LC over other launchers...
when is LC 12 gonna get the options back?

i know its a slow road back

**Comment by SuperDragonXD** [comment, marked as answer]
Hello. I know that this is a bit late, but we now support app drawer folders in Lawnchair 15+. App drawer tabs are tracked in #3147.

**Reply by Spoonmugen** [reply]
i'll scope it out - cheers mate
